### PR TITLE
Fixed the language option for the getTounaments method

### PR DESCRIPTION
**Pull request by theo-bnts**

### CODE REVIEW

File: lib/fortnite-api.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ class FortniteAPI {
      * @param {String} [options.lang]
      * @returns {Promise}
      */
-    getBattlepassRewards(season = "current", options = {}) {
+    getBattlepassRewards(options = {}, season = "current") {
```
Comment (benhawley7) [inline lib/fortnite-api.js]: I think this change is out of the scope of this pull request - it is also a breaking change.

**Review by benhawley7 — CHANGES_REQUESTED** [review]
Changes to the `getTournaments` method are fine, but I don't think we should be changing the `getBattlepassRewards` arguments order here.

File: lib/fortnite-api.js (hunk 1 above)
Comment (theo-bnts) [reply lib/fortnite-api.js]: Okay, I have resend the older version.

**Review by benhawley7 — APPROVED** [review]
Looks good! Thank you for contributing!

**Pull request merged**